gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/mapper/medication/MedicationRequestPlanMapperTest.java

```diff
@@ -613,12 +614,8 @@ private MedicationRequest getMedicationRequestFromEhrExtract(RCMRMT030101UKEhrEx
     }
 
     private void assertMetaSecurityNotPresent(MedicationRequest request) {
-        final Meta meta = request.getMeta();
 
-        assertAll(
-            () -> assertThat(meta.getSecurity()).isEmpty(),
-            () -> assertThat(meta.getProfile().getFirst().getValue()).isEqualTo(META_PROFILE)
-        );
+        assertMetaSecurityIsNotPresent(request.getMeta(), META_PROFILE);
```

Comment:
I think the methods `assertMetaSecurityIsPresent()` and `assertMetaSecurityIsNotPresent()` are redundant in favour of something like:

```java
assertThat(actualMeta).usingRecursiveComparison().isEqualTo(expectedMeta);
```